templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -566,6 +567,19 @@ export class {{ service.name }}Client {
 {%- endif %}
     return this._innerApiCalls.{{ method.name.toCamelCase() }}(request, options, callback);
   } 
+
+  {{ method.name.toCamelCase() }}Stream(
```

Comment:
We need to have a comments block above this method.

Reply:
I'm considering adding the comment in a separate pr, which may change the structure of the comments.

Reply:
OK!